docs/build/get-started.md

```diff
@@ -425,17 +425,17 @@ pub fn hello_world() -> Hello {        // 5
 }
 ```
 
-Let's review the code.
+#### Let's review the code.
 
-Before anything, (1) we need to import the [Marine Rust SDK](/docs/marine-book/marine-rust-sdk/marine-rust-sdk.md), which allows us to compile Rust code to wasm32-wasi module compatible with Fluence’s Marine runtime.
+1. Before anything, we need to import the [Marine Rust SDK](/docs/marine-book/marine-rust-sdk/marine-rust-sdk.md), which allows us to compile Rust code to wasm32-wasi module compatible with Fluence’s Marine runtime.
 
-In (2), we define a `Hello` structure that will hold our greeting message. We could also use just `-> String` for that matter, but this way you learn more. `#[marine]` macro marks a struct as publicly visible, and handles serialization/deserialization for you. It is a part of the *marine-rust-sdk*.
+2. (2) defines a `Hello` structure that will hold our greeting message. We could also use just `-> String` for that matter, but this way you learn more. `#[marine]` macro marks a struct as publicly visible, and handles serialization/deserialization for you. It is a part of the *marine-rust-sdk*.
```

Comment:
```suggestion
2. Defines a `Hello` structure that will hold our greeting message. We could also use just `-> String` but that wouldn't have allowed us to introduce the marine struct concept: `#[marine]` macro marks a struct as publicly visible, and handles serialization/deserialization for you. It is part of the *marine-rust-sdk*.
```
i light of the teachable moment, probably should explain  `pub` on struct fields.

Reply:
Well, I can remove `pub`, but it will produce a warning. That's Rust shenanigans caused by implicit serialization introduced by marine. 

It doesn't cause any effects per se, but is pretty complicated to explain to readers. Not sure we should spend their focus on that 🤔

Reply:
you're right. it's getting started. thee is a marine book after all.